ui/app/AppLayouts/Communities/panels/MembersTabPanel.qml

```diff
@@ -62,7 +65,33 @@ Item {
             Layout.fillWidth: true
             Layout.fillHeight: true
 
-            model: root.model
+            model: SortFilterProxyModel {
+                id: membersModel
+
+                sourceModel: root.model
+
+                function compare(modelLeft, modelRight) {
+                    const left = ProfileUtils.displayName(modelLeft.localNickname, modelLeft.ensName, modelLeft.displayName, modelLeft.alias);
+                    const right = ProfileUtils.displayName(modelRight.localNickname, modelRight.ensName, modelRight.displayName, modelRight.alias);
+
+                    if (left < right)
+                        return -1
+                    else if (left > right)
+                        return 1
+                    return 0;
+                }
+
+                sorters : [
+                    FastExpressionSorter {
+                        expression: {
+                            membersModel;
+                            root.model
+                            return membersModel.compare(modelLeft, modelRight)
+                        }
+                        expectedRoles: ["localNickname", "displayName", "ensName", "alias"]
+                    }
```

Comment:
This can be greatly simplified by creating an extra proxy role using `ProfileUtils.displayName()` as you do here, and then use a simple `StringSorter` on that extra proxy role

Reply:
Thanks for suggestion